apps/arweave/src/ar_merkle.erl

```diff
@@ -190,6 +196,112 @@ validate_path_strict_data_split(_ID, _Dest, _LeftBound, _RightBound, _Path, _Pat
 		_DataSize) ->
 	false.
 
+%% @doc Validate the given merkle path where any subtrees may have 0-based offset.
+validate_path2(ID, Dest, RightBound, _Path) when RightBound =< 0 ->
```

Comment:
Instead of `validate_path2` what do you think about `validate_rebased_path` or `validate_path_with_offset` or something simiar